src/groups/GroupActions.js

```diff
@@ -210,11 +212,24 @@ const doPutQueryFromState = (query, dispatch) => {
     .catch(error => dispatch({type: QUERYSET_SAVE_FAILED, error}));
 };
 /* xenia_package */
-const doMakeQueryFromStateAsync = _.debounce((query, dispatch)=>{
+const doMakeQueryFromStateAsync = _.debounce((query, dispatch, app, replace)=>{
   dispatch(requestQueryset());
+<<<<<<< HEAD
```

Comment:
would you mind fixing this merge conflict if you get a chance? I don't want to step on what you intended here.